jest.config.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  transform: { '\\.(js|jsx)?$': 'babel-jest' },
+  modulePathIgnorePatterns: [
+    '<rootDir>/build/',
+    '<rootDir>/public/',
+    '<rootDir>/dist/',
+    '<rootDir>/coverage/',
+    '<rootDir>/node_modules/',
+    '<rootDir>/index.js',
```

Comment:
This file is the main entry-point file in generator; it should not be under ignore.